docker/shared_steps/enable_php_modules.sh

```diff
@@ -1,12 +1,13 @@
 #!/bin/sh
 
-[ -z "$PHP_EXTENSIONS_STARTUP_ENABLE" ] || return 0
+[ ! -z "${PHP_EXTENSIONS_STARTUP_ENABLE}" ] || return 0
 
 for ext in $PHP_EXTENSIONS_STARTUP_ENABLE; do
     sectionText "Enable PHP extension $ext"
-    docker-php-ext-enable $ext
-    if [ -e "/usr/local/etc/php/mods-available/$ext.ini" ]; then
-        ln -fs /usr/local/etc/php/mods-available/$ext.ini /usr/local/etc/php/conf.d/20-$ext.ini
+    normalized_ext=`echo "$ext" | cut -d- -f1 | tr '[:upper:]' '[:lower:]'`
+    docker-php-ext-enable $normalized_ext
+    if [ -e "/usr/local/etc/php/mods-available/$normalized_ext.ini" ]; then
```

Comment:
doesn't do `docker-php-ext-enable` exactly the same?

Reply:
Please indicate what exactly you mean ...

Reply:
in line 9-11 you create a symlink in order to enable the php extension. I thought that this is already done by the command in line 8